package_manifest.yaml

```diff
@@ -246,6 +246,7 @@ ruby_core_packages:
 
 ruby_core_packages_tier1:
   hosts:
+    libsass: {}
```

Comment:
feels a bit weird to have it in `ruby_core_packages`, but then again, it's needed for ruby things…